### PR TITLE
gross-718. Add Hive URL policies tests for Spark/Trino

### PR DESCRIPTION
**Pull request by xBis7**
https://github.com/G-Research/gr-oss/issues/718

### CODE REVIEW

**Comment by xBis7** [comment]
This PR is ready for review but it's a draft because it's depending on https://github.com/xBis7/docker-setup-helper-scripts/pull/33 and until the ranger api PR is merged, this can't move forward.

File: compose/hive/conf/ranger-hive-security_hive_url_policies_enabled.xml [hunk 1]
```diff
@@ -89,9 +89,9 @@
 
 	<property>
```
Comment (mladjan-gadzic) [inline compose/hive/conf/ranger-hive-security_hive_url_policies_enabled.xml:90]: If I remember correctly, this property is skipped when Hive URL policies are enabled. Do we need it at all?

File: trino-hms-hdfs-ranger/tests/hive_url_policies_env_setup.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```
Comment (mladjan-gadzic) [inline trino-hms-hdfs-ranger/tests/hive_url_policies_env_setup.sh:1]: What is the difference between this and setup_for_testing_hms_url_policies.sh script?

**Comment by mladjan-gadzic** [comment]
All tests are passing. Let me know about previous comments. Otherwise LGTM.

File: compose/hive/conf/ranger-hive-security_hive_url_policies_enabled.xml (hunk 1 above)
Comment (xBis7) [reply compose/hive/conf/ranger-hive-security_hive_url_policies_enabled.xml:90]: You are right. I cleaned iit up.

File: trino-hms-hdfs-ranger/tests/hive_url_policies_env_setup.sh (hunk 2 above)
Comment (xBis7) [reply trino-hms-hdfs-ranger/tests/hive_url_policies_env_setup.sh:1]: `hive_url_policies_env_setup.sh` is used for setting up the environment for the hive url tests.

`setup_for_testing_hms_url_policies.sh` isn't used by any script. It's a minimal setup that I use a lot for manual testing and I thought about pushing it.

Comment (xBis7) [reply trino-hms-hdfs-ranger/tests/hive_url_policies_env_setup.sh:1]: I'll add comments to both of them.

**Review by mladjan-gadzic — APPROVED** [review]

**Pull request merged**